src/tests/Common/Coreclr.TestWrapper/MobileAppHandler.cs

```diff
@@ -41,120 +35,119 @@ private static int HandleMobileApp(string action, string platform, string catego
             using (var outputWriter = new StreamWriter(outputStream))
             using (var errorWriter = new StreamWriter(errorStream))
             {
-                if ((platform != "android") && (platform != "apple"))
-                {
-                    outputWriter.WriteLine($"Incorrect value of platform. Provided {platform}. Valid strings are android and apple.");
-                    platformValueFlag = false;
-                }
-
-                if ((action != "install") && (action != "uninstall"))
+                if (File.Exists($"{testBinaryBase}/.retry"))
```

Comment:
One more thing I don't think is great is that now the code is full of this `$"{testBinaryBase}/.retry"`.
Another thing is that we have these blocks of code "check exit code, log, ask for retry" that are also repetitive.

It would make much more sense to have a 2 or 3 methods on this class (or the other, wherever it's easier to log):
- RequestRetry
- IsRetryRequested
- IsRetryNeeded (to check for the retriable codes)

Or possibly just 2 to not have the repetitive blocks:
- IsRetryRequested
- HandleExitCode / DetectRetry

Again, I am worried someone would move this `.retry` or change `testBinaryBase` but only on one side and when stuff like this breaks, we will never learn about it